### ISSUE TITLE
error: Value for SWIFT_VERSION cannot be empty. (in target 'CocoaLumberjackSwift')

### ISSUE THREAD

**Issue by FatChou** [CocoaLumberjack/CocoaLumberjack#988]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [x] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | osx
 Platform Version        | 10.14
 CocoaLumberjack Version | Newest
 Integration Method      | carthage
 Xcode Version           | Xcode 10.1
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
# carthage update --platform macOS
<details><summary>build error log</summary><pre>
/usr/bin/xcrun xcodebuild -workspace /Users/pop/Documents/VPN/Specht/Carthage/Checkouts/CocoaLumberjack/Framework/Lumberjack.xcworkspace -scheme CocoaLumberjackSwift -configuration Release -derivedDataPath /Users/pop/Library/Caches/org.carthage.CarthageKit/DerivedData/10.1_10B61/CocoaLumberjack/2.3.0 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/dz/s6p42lmd62x7d7sk4h06h1z40000gn/T/CocoaLumberjack SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/pop/Documents/VPN/Specht/Carthage/Checkouts/CocoaLumberjack)User defaults from command line:
    IDEArchivePathOverride = /var/folders/dz/s6p42lmd62x7d7sk4h06h1z40000gn/T/CocoaLumberjack
    IDEDerivedDataPathOverride = /Users/pop/Library/Caches/org.carthage.CarthageKit/DerivedData/10.1_10B61/CocoaLumberjack/2.3.0

Build settings from command line:
    CARTHAGE = YES
    CLANG_ENABLE_CODE_COVERAGE = NO
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO
    GCC_INSTRUMENT_PROGRAM_FLOW_ARCS = NO
    ONLY_ACTIVE_ARCH = NO
    SKIP_INSTALL = YES
    STRIP_INSTALLED_PRODUCT = NO

note: Using new build system
note: Planning build
note: Constructing build description
Build system information
error: Value for SWIFT_VERSION cannot be empty. (in target 'CocoaLumberjackSwift')
</pre></details>

**Comment by ffried** [comment]
I've just tried to reproduce this with the following setup:
`Cartfile`:
```
github "CocoaLumberjack/CocoaLumberjack" "master"
```

`carthage update --platform macOS`:
```
*** Fetching CocoaLumberjack
*** Checking out CocoaLumberjack at "354ddbdcb7df3e2bbbde134b307449de7e3012fa"
*** xcodebuild output can be found in /var/folders/0z/101fmx1n5hl3g6l0vrn5g9qh0000gn/T/carthage-xcodebuild.yAOgtp.log
*** Building scheme "CocoaLumberjack" in Demos.xcworkspace
*** Building scheme "CocoaLumberjackSwift" in Demos.xcworkspace
```

It builds fine for me. It's strange, however, that it takes the Demos workspace for building. I assume because it's the first workspace it finds walking through the file hierarchy (and it does so recursively it seems).

@FatChou What does your Cartfile look like? What tag / commit hash does CocoaLumberjack resolve to when running `carthage update`?

**Comment by lolgear** [comment]
@ffried 
> It builds fine for me. It's strange, however, that it takes the Demos workspace for building.

It is definitely a bug in project or in Readme.

**Comment by FatChou** [comment]
In the main project, only github "zhuhaow/NEKit" "0.3.0" is used, it should be that his library depends on your project. @ffried
